tools/perfscale-audit/metric-client/metric-client.go

```diff
@@ -50,15 +50,15 @@ const (
 //	at all times.
 const (
 	vmiPhaseCount                 = `sum by (phase) (kubevirt_vmi_phase_count{})`
-	avgVirtAPICPUUsage            = `avg(sum(rate(container_cpu_usage_seconds_total{namespace="kubevirt",pod=~"virt-api.*", container!="",container!="POD"}[%ds]))by (pod))`
+	avgVirtAPICPUUsage            = `avg(rate(container_cpu_usage_seconds_total{namespace="kubevirt",pod=~"virt-api.*", container!="",container!="POD"}[%ds]))`
```

Comment:
the avgVirtAPIMemUsageInMB uses `avg(avg_over_time())` why does this not use the same `avg(avg_over_time(rate()))`?

Reply:
container_memory_rss is a gauge metric, it is giving the memory usage at a given point in time. rate() is used in cases of counter metrics where data is accumulated over time. avg(avg_over_time()) is more appropriate in this case.